README.en.md

```diff
@@ -0,0 +1,76 @@
+# Open Zaak
+
+*Production-ready APIs for case management (zaakgericht werken).*
+
+[Read this in Dutch](readme.rst).
+
+| | | |
+|-|-|-|
+| **Version:** | 1.2.0 |
```

Comment:
This file will have to be added to the [bumpversion config](https://github.com/open-zaak/open-zaak/blob/master/.bumpversion.cfg#L6)

Reply:
Done in 7eff1c568ac67873a12fa93505fee8834ee6f439